### ISSUE TITLE
Rest API to start timer and enter time

### ISSUE THREAD

**Issue by chhil** [tick/tick-api#41]
Is there a rest call to start a timer for a task  and another to stop/enter time based on the timer?

**Comment by jjjpppp** [comment]
@chhil Yes there is, we need to add it to the documentation. You can call all normal RESTful actions but ill list a few below for you to get you started

**First create a timer**: POST /:subscription_id/api/v2/timers
    - send params as { timer: { task_id: 1, notes: "starting timer" }} 
**To start a timer:** PUT /:subscription_id/api/v2/timers/:id/start
**To pause a timer**: PUT /:subscription_id/api/v2/timers/:id/pause

**Comment by chhil** [comment]
Thank you. Will give it a try.
